### DISCUSSION TITLE
Missing Metrics from GitHub Copilot API Since November 25, 2024

### DISCUSSION THREAD

**Discussion by jlopezcgl** [community/community#145783] [Copilot Conversations]
Hi,

We’re experiencing an issue where GitHub Copilot API metrics have not been available since November 25, 2024. The last day we have recorded data for is November 24, 2024, but no metrics have been generated or retrieved after that date.

Details of the Issue:
Expected behavior: Metrics should be available daily, including after November 24, 2024.
Actual behavior: No data is available from November 25 onwards.
Steps taken so far:
Verified API request functionality.
Checked for potential changes in the API documentation or behavior.
Confirmed no internal changes were made to our usage of the API.
Request for Assistance:
We’re reaching out to see if:

Anyone else is experiencing this issue.
There’s been a recent update to the API that we might have missed.
Any suggestions or workarounds exist to resolve this.
If this is related to a known issue or there’s documentation we should check, we’d greatly appreciate guidance.

Thank you in advance for your help! 🙏

**Comment by massimogentilini** [comment]
same here

**Comment by tkhieu** [comment]
Same here. We are running an AI adoption program and need this API for tracking but got 0 values for last week metric

**Comment by fviken** [comment]
Same issue in our company.

**Comment by dstaudenmaier** [comment]
Same issue in our company

**Comment by massimogentilini** [comment]
Seems solved but misses all data from 24/11 up to yesterday. For all the ones following I'm opening another issue, please vote it.

**Reply by tkhieu** [reply]
Yeah, 24th Nov then jump to 1st Dec. I really hope they can fix the lost data. Or I have to say to the chairman that I will extend this week report to next week and accept data lost

**Reply by massimogentilini** [reply]
https://github.com/orgs/community/discussions/146058
